apps/client/src/flogo/home/apps-list/apps-list.component.ts

```diff
@@ -68,7 +60,6 @@ export class FlogoAppsListComponent implements OnInit {
     } catch (error) {
       this.notifyUser(false, error);
     }
-    this.importInput.nativeElement.value = '';
```

Comment:
Just one thing, we needed this functionality as it would prevent importing the same application json again.